src/index.html

```diff
@@ -1,13 +1,55 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+       src="images/imac.jpeg"
+       alt="product image"
+      >
```

Comment:
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/83400b57-239a-4713-aed2-7be67670b338)

- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/59cd90f7-7384-4648-a374-cb73fa7850cc)

Fix in all places

Reply:
This is not done yet.